pkg/downloader/manager.go

```diff
@@ -85,6 +85,10 @@ type Manager struct {
 // If SkipUpdate is set, this will not update the repository.
 func (m *Manager) Build() error {
 	c, err := m.loadChartDir()
+	if c.Metadata.Dependencies == nil && m.Debug {
```

Comment:
I think this should be done around line 101 (after `req := c.Metadata.Dependencies`). And certainly after the error check.

Right now, the change here would ignore `err` and probably simply print `"No dependencies found to build"`, which would be confusing at best.

Reply:
I have moved to after  the error check. But if I move it to after req := `c.Metadata.Dependencies`. 
```
	lock := c.Lock
	if lock == nil {
		return m.Update()
	}
```
We may end up returning here and still not print anything